### DISCUSSION TITLE
Probleme mit Vercel beim React Calender Styling

### DISCUSSION THREAD

**Discussion by therealchrismcfly** [neuefische/questions#143] [Web] [answered]
Guten Morgen,

ich habe in meinem Projekt mit react-calendar gearbeitet. Um ihn zu stylen habe ich folgendes gemacht: 

![Bildschirmfoto 2022-09-08 um 11 26 17](https://user-images.githubusercontent.com/107792731/189086759-c8334400-5a2e-4987-8bfd-21e7aa824709.png)

Quelle: https://blog.logrocket.com/react-calendar-tutorial-build-customize-calendar/

In meinem Localhost sieht alles gut aus, so wie ich mir das wünsche: 
<img width="370" alt="Bildschirmfoto 2022-09-08 um 11 22 06" src="https://user-images.githubusercontent.com/107792731/189086893-8f430b9e-affb-4a9b-a953-f77fa0f5ff15.png">

Wenn ich deploye leider nicht mehr. Es wird hier nur ein Teil meiner Änderungen angezeigt:
<img width="383" alt="Bildschirmfoto 2022-09-08 um 11 22 16" src="https://user-images.githubusercontent.com/107792731/189086946-6e1c13be-ef8b-4d34-82c1-881ff6d46df3.png">

Hat jemand eine Idee, woran das liegen könnte und wie ich das fixen kann?

Hier der Link zum aktuellen PR: https://github.com/therealchrismcfly/capstone-project/pull/37

Danke schonmal :-)

**Comment by ahohnsen** [comment, marked as answer]
Hallo Chris,

Soweit ich es sehen kann, hast du (wie in der Quelle angegeben) die Styles aus der `Calendar.css` in den `node_modules` in deine `GlobalStyles` kopiert. Allerdings hast du dort, wo du den Kalender verwendest, weiterhin die `Calendar.css` aus dem `node_modules` Ordner importiert. Diesen Import solltest du löschen, da du ja jetzt die Styles aus deiner `GlobalStyles` Datei verwenden möchtest. 
Ich würde dir übrigens empfehlen, die Styles in eine separate Datei zu kopieren und dann diese Datei in deiner Kalender Komponente zu importieren. Dann ist deine `GlobalStyles` Datei etwas aufgeräumter. 🙂
